src/writer.jl

```diff
@@ -66,18 +66,16 @@ _print(io::IO, arr::AbstractVector, level::Int=0, ignore_level::Bool=false) =
 
 # print a single key-value pair
 function _print(io::IO, pair::Pair, level::Int=0, ignore_level::Bool=false)
-    key = if pair[1] === nothing
-        "null" # this is what the YAML parser interprets as 'nothing'
-    else
-        string(pair[1]) # any useful case
-    end
+    p0 = first(pair)
+    p1 = last(pair)
```

Comment:
These names are not so important but I would have gone for `p1` and `p2` rather than `p0` and `p1`.